File: Modul-3/C/PRAK304-2310817110006-Adrian_Bintang_Saputera.c
#include <stdio.h>

int main()
{
  int angka;

  printf("Masukkan sebuah angka (0-100): ");
  scanf("%d", &angka);

  if (angka == 0)
  {
    printf("Nol\n");
  }
  else if (angka >= 1 && angka <= 9)
  {
    printf("Satuan\n");
  }
  else if (angka >= 10 && angka <= 19)
  {
    printf("Belasan\n");
  }
  else if (angka >= 20 && angka <= 99)
  {
    printf("Puluhan\n");
  }
  else if (angka == 100)
  {
    printf("Anda Menginput Melebihi Limit Bilangan\n");
  }

  return 0;
}

File: Modul-3/Python/PRAK304-2310817110006-Adrian_Bintang_Saputera.py
angka = int(input("Masukkan sebuah angka (0-100): "))

if angka == 0:
    print("Nol")
elif angka >= 1 and angka <= 9:
    print("Satuan")
elif angka >= 10 and angka <= 19:
    print("Belasan")
elif angka >= 20 and angka <= 99:
    print("Puluhan")
elif angka == 100:
    print("Anda Menginput Melebihi Limit Bilangan")